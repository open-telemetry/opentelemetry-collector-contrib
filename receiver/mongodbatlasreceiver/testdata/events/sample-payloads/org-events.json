[
    {
        "created": "2018-07-12T16:30:05Z",
        "eventTypeName": "JOINED_TEAM",
        "id": "b3ad04e680eef540be141abe",
        "isGlobalAdmin": true,
        "links": [
            {
                "href": "https://cloud.mongodb.com/api/atlas/v1.0/orgs/5b478b3afc4625789ce616a3/events/5b47820d80eef5264e12514a",
                "rel": "self"
            }
        ],
        "orgId": "5b478b3afc4625789ce616a3",
        "remoteAddress": "203.0.113.22",
        "targetUsername": "b.doe@example.com",
        "userId": "5898b79080eef53b3ad04e68",
        "username": "j.doe@example.com"
    },
    {
        "created": "2018-07-09T21:14:40Z",
        "eventTypeName": "GROUP_CREATED",
        "groupId": "5b43d04087d9d6357de591a2",
        "id": "5b478b3afc49d6357de591af",
        "isGlobalAdmin": false,
        "links": [
            {
                "href": "https://cloud.mongodb.com/api/atlas/v1.0/orgs/5b478b3afc4625789ce616a3/events/5b43d04087d9d6357de591af",
                "rel": "self"
            }
        ],
        "orgId": "5b478b3afc4625789ce616a3",
        "remoteAddress": "192.0.2.88",
        "userId": "5898b79080eef53b3ad04e68",
        "username": "j.doe@example.com"
    }
]
