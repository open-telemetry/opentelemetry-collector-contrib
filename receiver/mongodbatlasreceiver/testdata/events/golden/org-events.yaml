resourceLogs:
  - resource:
      attributes:
        - key: mongodbatlas.org.id
          value:
            stringValue: test-org-id
    scopeLogs:
      - logRecords:
          - attributes:
              - key: event.domain
                value:
                  stringValue: mongodbatlas
              - key: type
                value:
                  stringValue: JOINED_TEAM
              - key: id
                value:
                  stringValue: b3ad04e680eef540be141abe
              - key: group.id
                value:
                  stringValue: ""
              - key: user.name
                value:
                  stringValue: j.doe@example.com
              - key: target.user.name
                value:
                  stringValue: b.doe@example.com
              - key: user.id
                value:
                  stringValue: 5898b79080eef53b3ad04e68
              - key: remote.ip
                value:
                  stringValue: 203.0.113.22
            body:
              stringValue: '{"alertId":"","alertConfigId":"","created":"2018-07-12T16:30:05Z","eventTypeName":"JOINED_TEAM","hostname":"","id":"b3ad04e680eef540be141abe","isGlobalAdmin":true,"links":[{"rel":"self","href":"https://cloud.mongodb.com/api/atlas/v1.0/orgs/5b478b3afc4625789ce616a3/events/5b47820d80eef5264e12514a"}],"orgId":"5b478b3afc4625789ce616a3","remoteAddress":"203.0.113.22","targetUsername":"b.doe@example.com","userId":"5898b79080eef53b3ad04e68","username":"j.doe@example.com"}'
            observedTimeUnixNano: "1680099677582009000"
            spanId: ""
            timeUnixNano: "1531413005000000000"
            traceId: ""
        scope: {}
      - logRecords:
          - attributes:
              - key: event.domain
                value:
                  stringValue: mongodbatlas
              - key: type
                value:
                  stringValue: GROUP_CREATED
              - key: id
                value:
                  stringValue: 5b478b3afc49d6357de591af
              - key: group.id
                value:
                  stringValue: 5b43d04087d9d6357de591a2
              - key: user.name
                value:
                  stringValue: j.doe@example.com
              - key: user.id
                value:
                  stringValue: 5898b79080eef53b3ad04e68
              - key: remote.ip
                value:
                  stringValue: 192.0.2.88
            body:
              stringValue: '{"alertId":"","alertConfigId":"","created":"2018-07-09T21:14:40Z","eventTypeName":"GROUP_CREATED","groupId":"5b43d04087d9d6357de591a2","hostname":"","id":"5b478b3afc49d6357de591af","links":[{"rel":"self","href":"https://cloud.mongodb.com/api/atlas/v1.0/orgs/5b478b3afc4625789ce616a3/events/5b43d04087d9d6357de591af"}],"orgId":"5b478b3afc4625789ce616a3","remoteAddress":"192.0.2.88","userId":"5898b79080eef53b3ad04e68","username":"j.doe@example.com"}'
            observedTimeUnixNano: "1680099677582009000"
            spanId: ""
            timeUnixNano: "1531170880000000000"
            traceId: ""
        scope: {}
